Chat_App/Chat_App/UI/MessageTableViewCell.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  MessageTableViewCell.swift
+//  Chat_App
+//
+//  Created by Gi Oo on 01.05.23.
+//
+
+import UIKit
+
+final class MessageTableViewCell: UITableViewCell {
+
+  let containerView = UIView()
+  let messageLabel = UILabel()
+  let timeLabel = UILabel()
```

Comment:
I believe that all these properties should be made private to limit their access